libraries/AP_HAL_ChibiOS/hwdef/scripts/addfunc_parse.py

```diff
@@ -28,7 +28,7 @@ def is_pin(str):
         return False
 
 def parse_adc_table(fname, table):
-    csvt = csv.reader(open(fname,'rb'))
```

Comment:
I think you should remove this drive-by

Reply:
I don't consider this to be drive by given the script doesn't run without the change, and you can't generate the table without the script.